src/SelectionD3.re

```diff
@@ -0,0 +1,28 @@
+type t;
```

Comment:
What's the reason for having both this file and `Selection.re`?

Reply:
I forgot to delete Selection.re